### ISSUE TITLE
aria support for table paginator

### ISSUE THREAD

**Issue by yerramshilpa** [primefaces/primeng#11895]
### Describe the bug

Current behavior
1) When checking with the accessibility tool first/previous/next/last buttons do not have [Button has no accessible name, innerText, or [title].and Listbox Element has no accessible name.]for the page selection

2) When the custom header/header, the role is missing for the table row

Expected behavior
The screen reader should speak (vi aria-label) and the accessibility tool should not show error messages. when tabbed to each control "First", "Previous", "Next", "Last". When tabbing to the items per page dropdown a focus state should be shown to indicate the current control.

Minimal reproduction of the problem with instructions
Aria-label missing for the buttons.
https://github.com/primefaces/primeng/blob/19df5df79629a72490add7d078a8d0e5ce52f41e/src/app/components/paginator/paginator.ts

### Environment

https://github.com/primefaces/primeng/blob/19df5df79629a72490add7d078a8d0e5ce52f41e/src/app/components/paginator/paginator.ts

PrimeNg version :  "primeng": "13.1.0".

### Angular version

13.2.0

### PrimeNG version

13.1.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

8.1.0

![image](https://user-images.githubusercontent.com/44271677/189964532-f6b9d5cd-675a-4dfc-89b5-a1eaf2d8b742.png)
